src/plugins/intel_vpu/common/src/ngraph/transformations/convert_transpose_presicion.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "vpu/ngraph/transformations/convert_transpose_presicion.hpp"
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/opsets/opset7.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(vpu::ConvertTransposePrecision, "ConvertTransposePrecision", 0);
+using namespace std;
+
+namespace vpu {
+
+ConvertTransposePrecision::ConvertTransposePrecision() {
+    auto transpose_pattern = ngraph::pattern::wrap_type<ngraph::opset8::Transpose>();
+
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher& m) {
+        auto transpose_node = std::dynamic_pointer_cast<ngraph::opset8::Transpose>(m.get_match_root());
+        if (!transpose_node)
+            return false;
+
+        auto dst_type = ngraph::element::Type_t::f16;
+        auto src_type = ngraph::element::Type_t::u8;
```

Comment:
Just to clarify, do we need this transformation only for cases with U8 input?

Reply:
It was this case that broke the hello_classification sample for the Myriad plugin. So yes, that's all we need for now.